frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -97,12 +97,17 @@ class ChartSettings extends Component {
       this.setState({
         currentSection: (initial && initial.section) || null,
         currentWidget: (initial && initial.widget) || null,
+        widgetOverride: null,
```

Comment:
We are introducing the concept of `widgetOverride` with `writeSettingId` to the shared settings code only for one specific place. Would it be possible to still have the original settings definition but its widget would be a component that switches between two views?